argocd/ctrl_plane/dev/manifests/vmauth-tls.yaml

```
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: vmauth-tls
+  namespace: observability
+type: kubernetes.io/tls
+stringData:
+  tls.key: ""
+  tls.crt: ""
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)